src/WeekView/WeekView.js

```diff
@@ -93,13 +93,14 @@ export default class WeekView extends Component {
       );
     }
     if (
-      moment().isSame(this.props.selectedDate, 'date') ||
-      moment().isBetween(
-        this.props.selectedDate,
-        moment(this.props.selectedDate).add(this.props.numberOfDays, 'days'),
-        undefined,
-        '[)',
-      )
+      this.props.selectedDate !== prevProps.selectedDate &&
+      (moment().isSame(this.props.selectedDate, 'date') ||
+        moment().isBetween(
+          this.props.selectedDate,
+          moment(this.props.selectedDate).add(this.props.numberOfDays, 'days'),
+          undefined,
+          '[)',
+        ))
```

Comment:
p3; 괄호끼리의 indent는 가급적이면 맞춰주세요!
```suggestion
        )
      )
```